spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -234,6 +237,30 @@ def test_llm_logs_default_null_handler(nlp: Language, capsys: pytest.CaptureFixt
     assert f"LLM response for doc: {doc.text}" not in captured.out
 
 
+def test_fewshot_reader():
+    """Test if fewshot reader copes with file formats as expected."""
+    example = [
+        {
+            "text": "Circe lived on Aeaea.",
+            "entities": {"PER": ["Circe"], "LOC": ["Aeaea"]},
+        }
+    ]
+    with make_tempdir() as tmpdir:
+        srsly.write_yaml(tmpdir / "example.yml", example)
+        srsly.write_yaml(tmpdir / "example.json", example)
+        srsly.write_yaml(tmpdir / "example.foo", example)
+
+        warning = (
```

Comment:
Don't necessarily need the entire warning just to match on an obvious part.

Reply:
Shortened the warning.